### ISSUE TITLE
bug: callWithHighValueAndGasOOG.json error when loading test case file

### ISSUE THREAD

**Issue by khaeljy** [kkrt-labs/ef-tests#257]
# Bug Report

**Kakarot version:**

Kakarot sha 7686fbf022d99ce01927eb488ad59f0a0ec7ce5d

**Current behavior:**

When running stCallCreateCallCodeTest

```
thread 'blockchain_tests::st_call_create_call_code_test' panicked at 'test case should load: Io { path: "/workspaces/ef-tests/crates/ef-testing/ethereum-tests/GeneralStateTests/stCallCreateCallCodeTest/callWithHighValueAndGasOOG.json", error: "invalid length 18, expected a (both 0x-prefixed or not) hex string or byte array containing between (0; 8] bytes at line 1 column 6249" }', /workspaces/ef-tests/crates/ef-testing/src/traits/mod.rs:56:58
stack backtrace:
   0:     0x55ae385f1551 - std::backtrace_rs::backtrace::libunwind::trace::hd102c46c16771587
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/../../backtrace/src/backtrace/libunwind.rs:93:5
   1:     0x55ae385f1551 - std::backtrace_rs::backtrace::trace_unsynchronized::h6545cf7d6ed27551
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x55ae385f1551 - std::sys_common::backtrace::_print_fmt::h258fac35947dded0
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/sys_common/backtrace.rs:65:5
   3:     0x55ae385f1551 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hf070a48cb28a24d1
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/sys_common/backtrace.rs:44:22
   4:     0x55ae3861e14f - core::fmt::rt::Argument::fmt::h0a093c27aabe85c6
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/fmt/rt.rs:138:9
   5:     0x55ae3861e14f - core::fmt::write::h809e1a6b668324f6
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/fmt/mod.rs:1094:21
   6:     0x55ae385ed037 - std::io::Write::write_fmt::h2ed31d529426c73c
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/io/mod.rs:1713:15
   7:     0x55ae385f1365 - std::sys_common::backtrace::_print::h72058f630d074235
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/sys_common/backtrace.rs:47:5
   8:     0x55ae385f1365 - std::sys_common::backtrace::print::h71140e0244457266
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/sys_common/backtrace.rs:34:9
   9:     0x55ae385f2fc3 - std::panicking::default_hook::{{closure}}::h4cc812472591f5d0
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panicking.rs:269:22
  10:     0x55ae385f2d54 - std::panicking::default_hook::h63cd13622fb98d85
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panicking.rs:288:9
  11:     0x55ae385f3549 - std::panicking::rust_panic_with_hook::h521173c88e8eb80d
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panicking.rs:705:13
  12:     0x55ae385f3447 - std::panicking::begin_panic_handler::{{closure}}::he04e6c92307b76ce
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panicking.rs:597:13
  13:     0x55ae385f1986 - std::sys_common::backtrace::__rust_end_short_backtrace::hb3ba44a9896576c5
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/sys_common/backtrace.rs:151:18
  14:     0x55ae385f3192 - rust_begin_unwind
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panicking.rs:593:5
  15:     0x55ae359fef23 - core::panicking::panic_fmt::h69460e28c51049e7
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/panicking.rs:67:14
  16:     0x55ae359ff4b3 - core::result::unwrap_failed::h5a2d7db84920c322
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/result.rs:1651:5
  17:     0x55ae35a0e342 - core::result::Result<T,E>::expect::h4ea7578e9885503b
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/result.rs:1033:23
  18:     0x55ae35a0eb97 - ef_testing::traits::Suite::run::{{closure}}::h7207cacdf4d32f3b
                               at /workspaces/ef-tests/crates/ef-testing/src/traits/mod.rs:56:24
  19:     0x55ae365e4e39 - <core::pin::Pin<P> as core::future::future::Future>::poll::h4a9e245035bf96e0
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/future/future.rs:125:9
  20:     0x55ae35a04886 - tests::blockchain_tests::st_call_create_call_code_test::{{closure}}::h8aaf1307b1d0ddd8
                               at /workspaces/ef-tests/crates/ef-testing/tests/tests.rs:74:5
  21:     0x55ae35a08998 - <core::pin::Pin<P> as core::future::future::Future>::poll::hfcb1fad841c0aef6
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/future/future.rs:125:9
  22:     0x55ae35a12834 - tokio::runtime::park::CachedParkThread::block_on::{{closure}}::h25f2752d89fea019
                               at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.32.0/src/runtime/park.rs:282:63
  23:     0x55ae35a1220a - tokio::runtime::coop::with_budget::h609f8d47840e8f9a
                               at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.32.0/src/runtime/coop.rs:107:5
  24:     0x55ae35a1220a - tokio::runtime::coop::budget::he4c9c1a1448153e2
                               at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.32.0/src/runtime/coop.rs:73:5
  25:     0x55ae35a1220a - tokio::runtime::park::CachedParkThread::block_on::h066bdb3c560f5f87
                               at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.32.0/src/runtime/park.rs:282:31
  26:     0x55ae35a05964 - tokio::runtime::context::blocking::BlockingRegionGuard::block_on::h486e7c6815d76e0a
                               at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.32.0/src/runtime/context/blocking.rs:66:9
  27:     0x55ae35a0f218 - tokio::runtime::scheduler::multi_thread::MultiThread::block_on::{{closure}}::h0fb48fbd6dee6c73
                               at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.32.0/src/runtime/scheduler/multi_thread/mod.rs:87:13
  28:     0x55ae35a12ff4 - tokio::runtime::context::runtime::enter_runtime::hc0294d1035c2b407
                               at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.32.0/src/runtime/context/runtime.rs:65:16
  29:     0x55ae35a0f1e8 - tokio::runtime::scheduler::multi_thread::MultiThread::block_on::h7de2632070888111
                               at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.32.0/src/runtime/scheduler/multi_thread/mod.rs:86:9
  30:     0x55ae35a00ca7 - tokio::runtime::runtime::Runtime::block_on::h26457ddf2c7cf183
                               at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.32.0/src/runtime/runtime.rs:349:45
  31:     0x55ae35a09b67 - tests::blockchain_tests::st_call_create_call_code_test::h77c7bc73c717e32a
                               at /workspaces/ef-tests/crates/ef-testing/tests/tests.rs:55:13
  32:     0x55ae35a04647 - tests::blockchain_tests::st_call_create_call_code_test::{{closure}}::he957d8b77afe9620
                               at /workspaces/ef-tests/crates/ef-testing/tests/tests.rs:53:31
  33:     0x55ae35a0a335 - core::ops::function::FnOnce::call_once::h201c4a5cba82327b
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/ops/function.rs:250:5
  34:     0x55ae35a4db4f - core::ops::function::FnOnce::call_once::h47e1e41b3b25277c
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/ops/function.rs:250:5
  35:     0x55ae35a4db4f - test::__rust_begin_short_backtrace::h9a84848d030117a1
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/test/src/lib.rs:655:18
  36:     0x55ae35a196ec - test::run_test::{{closure}}::h9c8a63b040029c5b
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/test/src/lib.rs:646:30
  37:     0x55ae35a196ec - core::ops::function::FnOnce::call_once{{vtable.shim}}::hce2270657d52a69f
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/ops/function.rs:250:5
  38:     0x55ae35a4cae9 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::hcc2b58da913fd56f
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/alloc/src/boxed.rs:1993:9
  39:     0x55ae35a4cae9 - <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once::h4495d4d5987c5b09
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/panic/unwind_safe.rs:271:9
  40:     0x55ae35a4cae9 - std::panicking::try::do_call::h5c3dc9add79dd4ee
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panicking.rs:500:40
  41:     0x55ae35a4cae9 - std::panicking::try::h960fdcb15420b25f
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panicking.rs:464:19
  42:     0x55ae35a4cae9 - std::panic::catch_unwind::h4e4806ece2ba28e3
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panic.rs:142:14
  43:     0x55ae35a4cae9 - test::run_test_in_process::hd7716023fba5061c
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/test/src/lib.rs:678:27
  44:     0x55ae35a4cae9 - test::run_test::run_test_inner::{{closure}}::h707cc95baee367b8
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/test/src/lib.rs:572:39
  45:     0x55ae35a13828 - test::run_test::run_test_inner::{{closure}}::h1990cddca5aaa207
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/test/src/lib.rs:599:37
  46:     0x55ae35a13828 - std::sys_common::backtrace::__rust_begin_short_backtrace::ha943b79ab24b0f04
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/sys_common/backtrace.rs:135:18
  47:     0x55ae35a1976f - std::thread::Builder::spawn_unchecked_::{{closure}}::{{closure}}::hde453869494e208b
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/thread/mod.rs:529:17
  48:     0x55ae35a1976f - <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once::h04af7c91037a0fa6
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/panic/unwind_safe.rs:271:9
  49:     0x55ae35a1976f - std::panicking::try::do_call::h97c77250559b64df
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panicking.rs:500:40
  50:     0x55ae35a1976f - std::panicking::try::hdb0719a218852397
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panicking.rs:464:19
  51:     0x55ae35a1976f - std::panic::catch_unwind::ha18a9ca61544e8a9
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/panic.rs:142:14
  52:     0x55ae35a1976f - std::thread::Builder::spawn_unchecked_::{{closure}}::h1ab538a49d99f76f
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/thread/mod.rs:528:30
  53:     0x55ae35a1976f - core::ops::function::FnOnce::call_once{{vtable.shim}}::hde5590fe12c68406
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/core/src/ops/function.rs:250:5
  54:     0x55ae385f8635 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::h7557df74034f0cd4
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/alloc/src/boxed.rs:1993:9
  55:     0x55ae385f8635 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::ha799d8f85cbbc72f
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/alloc/src/boxed.rs:1993:9
  56:     0x55ae385f8635 - std::sys::unix::thread::Thread::new::thread_start::h88ab86e849d99fd7
                               at /rustc/3b2073f0762cff4d3d625bb10017e0ce4e7abe50/library/std/src/sys/unix/thread.rs:108:17
  57:     0x7f02f72fab43 - <unknown>
  58:     0x7f02f738bbb4 - __clone
  59:                0x0 - <unknown>
```

**Expected behavior:**

Test case file is loaded

**Steps to reproduce:**

Unskip `blockchain_tests!(st_call_create_call_code_test, stCallCreateCallCodeTest);` in tests.rs and run ef-tests

**Comment by greged93** [comment]
This is due to the fact that the test files loads a value > U64::MAX, which causes serde to fail, I increased the type of the value to U128 so it should be ok now, will fix this and add the test in #473

**Comment by greged93** [comment]
Completed by #473
